sos/plugins/networking.py

```diff
@@ -17,10 +18,18 @@ class Networking(Plugin):
     plugin_name = "networking"
     profiles = ('network', 'hardware', 'system')
     trace_host = "www.example.com"
-    option_list = [(
+    option_list = [
         ("traceroute", "collect a traceroute to %s" % trace_host, "slow",
-         False)
-    )]
+         False),
+        ("namespaces", "specific namespaces regex to be " +
+         "collected, namespaces should be separated by space " +
+         "as for example \"eth* ens2\"", "fast", ""),
+        ("inspect_namespaces", "define how many namespaces should be " +
+         "collected, 0 for unlimited. Works ONLY when namespaces are NOT " +
+         "defined", "slow", 0),
```

Comment:
I think this could be worded differently. E.G.

`Number of namespaces to collect, 0 for unlimited. Incompatible with the namespaces plugin option`

Reply:
Parameter description updated as suggested.